api/pom.xml

```diff
@@ -296,6 +296,12 @@
 			<groupId>org.junit.jupiter</groupId>
 			<artifactId>junit-jupiter-params</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.assertj</groupId>
+			<artifactId>assertj-core</artifactId>
```

Comment:
Was this required?

Reply:
Yes, this is required in unit tests

Reply:
What errors do you get when you do not have it?

Reply:
org.assertj gives more options in assertions. I have used it in the new unit tests.

Reply:
I would avoid adding new libraries at this stage.

Reply:
Let us also remove this change.

Reply:
Done